lib/helper.py

```diff
@@ -76,11 +76,12 @@ def get_dist():
             elif line.startswith("VERSION="):
                 try:
                     line = line.replace('"', '')
-                    dist_ver = re.findall("VERSION=(\S+)", line)[0]
-                    dist_ver = re.match('(\d*)(\.)*(\d+)', dist_ver).group()
+                    dist_ver = re.findall("VERSION=(\S+)", line)[0].lower().replace("-",".")
                 except:
                     pass
         fd.close()
+    if os.uname()[-1] == 'ppc64':
```

Comment:
Though the older code is carried over, I suggest doing it like:
```
if 'le' not in os.uname()[-1]:
    dist += 'be'
```

Reply:
@narasimhan-v I am fine with older code. Though just curious if there is a particular reason which I may be missing for this change?